# SLBP binding region analysis

This github repository contains an example analysis Rmarkdown using [DEWSeq](https://bioconductor.org/packages/release/bioc/html/DEWSeq.html) package. The input files for this analysis were generated using [htseq-clip](https://htseq-clip.readthedocs.io/en/latest/) package.

## Dataset

We use [SLBP](https://www.genecards.org/cgi-bin/carddisp.pl?gene=SLBP) eCLIP data generated by the ENCODE consortium using K562 human cell lines in this analysis (1). The experimental design used by the consortium consisted of two IP samples compared against one SMI sample.

The .bam alignment files were generated by the consortium by aligning the reads after processing to hg38 human genome assembly using STAR aligner (1). For additional details of the read processing steps used by the consortium, please refer to the links in the table below:

| Experiment accession | Bam file accession | Sample description |
| -------|:------------:|:--------:|
| [ENCSR483NOP](https://www.encodeproject.org/experiments/ENCSR483NOP/) | [ENCFF218ZEI](https://www.encodeproject.org/files/ENCFF218ZEI/) |IP sample 1|
| [ENCSR483NOP](https://www.encodeproject.org/experiments/ENCSR483NOP/) | [ENCFF511HSJ](https://www.encodeproject.org/files/ENCFF511HSJ/) |IP sample 2|
| [ENCSR950WBG](https://www.encodeproject.org/experiments/ENCSR950WBG/) | [ENCFF879UID](https://www.encodeproject.org/files/ENCFF879UID/) |SMI sample|

## Packages used in this analysis

* [htseq-clip](https://htseq-clip.readthedocs.io/en/latest/)
* [DEWSeq](https://bioconductor.org/packages/release/bioc/html/DEWSeq.html)

This repository contains three files:
* Analysis [Rmarkdown file](https://github.com/EMBL-Hentze-group/DEWSeq_analysis_helpers/blob/master/SLBP_example/SLBP_analysis.Rmd)
* Results in [html format](https://github.com/EMBL-Hentze-group/DEWSeq_analysis_helpers/blob/master/SLBP_example/SLBP_analysis.html)
* Results in [pdf format](https://github.com/EMBL-Hentze-group/DEWSeq_analysis_helpers/blob/master/SLBP_example/SLBP_analysis.pdf)

## References
1. Van Nostrand EL, Pratt GA, Shishkin AA, et al (2016) Robust transcriptome-wide discovery of RNA-binding protein binding sites with enhanced CLIP (eCLIP). Nat Methods 13:508â€“514
